frontend/src/components/Modals/ResetPassword.js

```diff
@@ -41,17 +40,27 @@ const styles = theme => ({
 });
 
 class SimpleModal extends React.Component {
-  state = {
-    open: true,
-    password: '',
-    showPassword: false,
-  };
+  constructor(props) {
+    super(props)
+    this.state = {
+      open: false
+    };
+  }
+  
+  UNSAFE_componentWillReceiveProps(e){
```

Comment:
if you wanna use componentWillRecieveProps, use the new version: 
static getDerivedStateFromProps(nextProps, prevState)
[](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops)
Will be out to date in version 17.0.0